_data/addresses.yml

```diff
@@ -1,78 +0,0 @@
-'Chris Jefferson': 'http://caj.host.cs.st-andrews.ac.uk/'
-'David Joyner': 'https://sites.google.com/site/wdjoyner/'
-'Stefan Kohl': 'https://www.gap-system.org/DevelopersPages/StefanKohl/'
-'Alexander Konovalov': 'http://blogs.cs.st-andrews.ac.uk/alexk/'
-'Marco Kuhlkamp': 'http://www.math.rwth-aachen.de/~Marco.Kuhlkamp'
-'Steve Linton': 'http://www.dcs.st-and.ac.uk/~sal'
-'Frank Luebeck': 'http://www.math.rwth-aachen.de/~Frank.Luebeck/'
-'Klaus Lux': 'http://math.arizona.edu/~klux/'
-'Gunter Malle': 'http://www.mathematik.uni-kl.de/~malle/en/'
-'Andrew Mathas': 'http://www.maths.usyd.edu.au:8000/u/mathas'
-'John McDermott': 'http://www-maths.mcs.st-andrews.ac.uk/homepages/jjm5.html'
-'Jean Michel': 'http://www.math.jussieu.fr/~jmichel/'
-'G. Michler': 'http://www.exp-math.uni-essen.de/people/michler.html'
-'James Mitchell': 'http://www-history.mcs.st-and.ac.uk/~jamesm/'
-'Robert F. Morse': 'http://faculty.evansville.edu/rm43'
-'Juergen Mueller': 'http://www.math.rwth-aachen.de/~Juergen.Mueller/'
-'Gabriele Nebe': 'http://www.math.rwth-aachen.de/~Gabriele.Nebe/index.html.en'
-'Joachim Neubueser': 'http://www.math.rwth-aachen.de/~Joachim.Neubueser/'
-'Max Neunhoeffer': 'http://www-groups.mcs.st-and.ac.uk/~neunhoef/index.html'
-'M. F. Newman': 'http://wwwmaths.anu.edu.au/~newman/'
-'Werner Nickel': 'http://www.mathematik.tu-darmstadt.de/~nickel'
-'Alice Niemeyer': 'https://wwwb.math.rwth-aachen.de/Mitarbeiter/niemeyer.php'
-'Christof Noebauer': 'http://www.algebra.uni-linz.ac.at/'
-'Richard Parker': 'http://web.ukonline.co.uk/richard'
-'Dmitrii Pasechnik': 'http://www.cs.ox.ac.uk/people/dmitrii.pasechnik/'
-'Goetz Pfeiffer': 'http://schmidt.ucg.ie/~goetz'
-'Markus Pfeiffer': 'http://www.morphism.de/~markusp'
-'Wilhelm Plesken': 'http://wwwb.math.rwth-aachen.de/Mitarbeiter/plesken.html'
-'Cheryl Praeger': 'http://www.maths.uwa.edu.au/~praeger'
-'Markus Pueschel': 'http://www.ece.cmu.edu/~pueschel'
-'Julianne Rainbolt': 'http://euler.slu.edu/Dept/Faculty/rainbolt/'
-'Colin Ramsay': 'http://www.csee.uq.edu.au/~cram'
-'Sarah Rees': 'http://www.mas.ncl.ac.uk/~nser/'
-'E. Robertson': 'http://www-groups.dcs.st-and.ac.uk:80/~edmund/'
-'Marc Roeder': 'http://www.maths.nuigalway.ie/~roeder/'
-'Gerhard Erich Roehrle': 'http://web.mat.bham.ac.uk/G.E.Roehrle/'
-'Colva Roney-Dougal': 'http://www.dcs.st-and.ac.uk/~colva/'
-'N. Ruskuc': 'http://www-groups.dcs.st-and.ac.uk/~nik/'
-'Jack Schmidt': 'http://www.ms.uky.edu/~jack/'
-'J. J. Seidel': 'http://www.win.tue.nl/win/math/dw/personalpages/seidel/'
-'Charles Sims': 'http://www.math.rutgers.edu/~sims/'
-'Leonard Soicher': 'http://www.maths.qmul.ac.uk/~leonard/'
-'Martin Schoenert': 'http://www.math.rwth-aachen.de/~Martin.Schoenert/'
-'Chris Wensley': 'http://www.bangor.ac.uk/~mas023/'
-'R. Wilson': 'http://for.mat.bham.ac.uk/R.A.Wilson/'
-'C. Wright': 'http://darkwing.uoregon.edu/~wright'
-'GAP': 'https://www.gap-system.org/Contacts/contacts.html'
-'Eamonn-O-Brien': 'http://www.math.auckland.ac.nz/~obrien'
```

Comment:
Eamonn is missing In the new .yaml and was hardcoded above somewhere. Why is he special? :D

Reply:
I added him back. But note that he was shown as "Eamonn-O-Brien" in the text, which was a regression; I wanted to fix it to show "Eamonn O'Brien" again, and did that before the rest of the namelink changes.

That said, he and his URL are "hard coded" in many other places already, there is a single "namelink" to him right now. We have to decide eventually whether to either use the "namelink" everywhere", or nowhere.

Reply:
Okay I added Eamonn and wasn't sure how to escape the `'` in his name. I didn't know that it would be nice to use the tags in the text.

I'd say we should use it everywhere eventually.